packages/contracts-bedrock/src/L1/LeaderElectionBatchInbox.sol

```diff
@@ -5,6 +5,8 @@ pragma solidity ^0.8.0;
 /// @notice Interface for implementing a leader election scheme
 
 abstract contract LeaderElectionBatchInbox {
+    uint256 public creation_block_number;
```

Comment:
Just noticed the variable naming style in the contracts we added is not in line with the project. They should be like `creationBlockNumber`. Don't have to fix it in this PR though we can make a separate one later.

Reply:
Sure, ticket created: https://github.com/EspressoSystems/op-leader-election/issues/74

Reply:
Good luck